.github/workflows/deploy-prod.yaml

```diff
@@ -6,6 +6,21 @@ on:
     - "v*.*.*"
 
 jobs:
+  verify-tag:
```

Comment:
Can you make the other jobs depend (`needs`) this job so that the workflow will not start if this fails.

Reply:
yes thanks. all of them depends on production-docker-image or kurl-util-image. Added in those.